### ISSUE TITLE
Should we move the BatchNorm + ReLu to the classification head?

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#1506]
https://github.com/keras-team/keras-cv/blob/fff847454802d343d3d8f5e6bf77296067d72283/keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py#L357

These are basically adding noise if we aren't including a classification top

**Comment by jbischof** [comment]
Sure I can do this in #1458
